src/components/ui/Suggest.jsx

```diff
@@ -159,3 +157,12 @@ Suggest.propTypes = {
 };
 
 export default Suggest;
+
+// Blur the focused field and close the suggest list when clicking outside of a field
+if (!window.onclick) {
+  window.onclick = e => {
```

Comment:
We should register the event with `addEventListeners` in order to allow multiple listeners

Reply:
well, I tried that at first, but the opposite happened: multiple event listeners were added, while this "onclick" function is always unique.

Reply:
yes, because the event listener should be registered once in a `useEffect`, (do not forget te cleanup function)